.changeset/cold-crabs-arrive.md

```diff
@@ -0,0 +1,6 @@
+---
+'@astrojs/markdown-remark': minor
+'astro': minor
+---
+
+Enable the "defaultColor" option from shiki
```

Comment:
```suggestion
Adds support for [Shiki's "defaultColor" option](https://shiki.style/guide/dual-themes#without-default-color).

This option allows you to override the values of a theme's CSS variables and use your own colors.

Configure `defaultColor: false` in your Shiki config to apply throughout your site, or pass to Astro's built-in `<Code>` component to style an individual code block.

```js title="astro.config.mjs"
import { defineConfig } from 'astro/config';
export default defineConfig({
  markdown: {
    shikiConfig: {
      themes: {
        light: 'github-light',
        dark: 'github-dark',
      },
      defaultColor: false,
    },
  },
});
```

```js
---
import { Code } from 'astro:components';
---
<Code code={`const useMyColors = true`} lang="js" defaultColor=false />
```

OK, thanks for waiting! I'm not sure whether all of this is entirely accurate, but this is the general style of message I'm thinking of here. So, using this as a model, can you make this accurate/appropriate? I think this would be a helpful way to introduce this new option!

Reply:
Thank you for the suggestion! I've added a couple small changes. I hope that is okay

Reply:
Yup, and I just updated some grammar and it's great, thank you!